src/gdcdictionary/schemas/_terms.yaml

```diff
@@ -7700,7 +7718,92 @@ hormonal_replacement_therapy_status:
     termDef:
       term: >-
         Hormone Replacement Therapy Status
+      source: NCIt
+      cde_id: C206479
+      cde_version: 24.07e
+      term_url: "https://ncit.nci.nih.gov/ncitbrowser/ConceptReport.jsp?dictionary=NCI_Thesaurus&ns=ncit&code=C206479"
+      term_id: C206479
+      term_version: 24.07e
+
+population_group:
+  common:
+    description: >-
+      A group of individuals united by a common factor (e.g., geographic location,
+      ethnicity, disease, age, gender)
```

Comment:
do we need to have a period at the end of the desc? (it's missing in the ticket too.)

Reply:
Great catch, added 👍